elements/rh-secondary-nav/rh-secondary-nav.ts

```diff
@@ -212,12 +212,17 @@ export class RhSecondaryNav extends LitElement {
   @bound
   private _focusOutHandler(event: FocusEvent) {
     const target = event.relatedTarget as HTMLElement;
-    if (target === null || (target && !target.closest('rh-secondary-nav'))) {
-      if (this._compact) {
-        this._mobileMenuExpanded = false;
+    if (target) {
```

Comment:
no `if { if`. use optional chaining

Reply:
Fixed in cd9062ab3f49fafe880c1f763b96f6c4ca81f52c